packages/ksf-registration/src/KSF/Registration/Component.purs

```diff
@@ -0,0 +1,498 @@
+module KSF.Registration.Component where
```

Comment:
Note that I decided to only have a `Component` which has all the `View` stuff as well. I think the separation (at least in this case) makes things unnecessarily complex (most of this code is view related anyway).